clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/caching/agents/TitusStreamingUpdateAgent.java

```diff
@@ -159,6 +159,16 @@ public String getProviderName() {
     return TitusCachingProvider.PROVIDER_NAME;
   }
 
+  @Override
+  public Collection<AgentDataType> getProvidedDataTypes() {
+    return TYPES;
+  }
+
+  @Override
+  public CacheResult loadData(ProviderCache providerCache) {
+    return null;
```

Comment:
I'd throw an explicit exception here instead. This method shouldn't get called, but if it did, would result in an odd NPE further up the stack.